### DISCUSSION TITLE
Please tell me your favorite cars

### DISCUSSION THREAD

**Discussion by Nagahama-toyo** [NAIST-SE/AttractingContributors#920] [General]
# Please tell me your favorite cars
I like sports cars. 

I want to know your favorite cars including non sports cars.

Please comment your favorite car's name, or information about it.

Repository:https://github.com/Nagahama-toyo/AttractingContributors-SportsCars

# あなたの好きな車を教えてください
スポーツカーが大好きなので、普通の車含めてあなたの好きな車を教えてもらえると嬉しいです

よろしくお願いします

リポジトリ:https://github.com/Nagahama-toyo/AttractingContributors-SportsCars

**Comment by rfLENtlr** [comment]
I like Camaro!!

Send a pull request.
Would you like to contribute to my repository as well?
https://github.com/rfLENtlr/AttractingContributors-OS

**Reply by Nagahama-toyo** [reply]
Thank you for your pull request.

I am looking forward to drive my car with your Camaro in the future.

I sent you a pull request as well.

**Comment by bconfessor** [comment]
Done! Could you please tell me your favorite song?? 
https://github.com/bconfessor/FavoriteSong/blob/main/README.md

**Comment by k-mizoguch1** [comment]
plz
https://github.com/kzu000/Food-what-do-you-like
